pipeline/schemas/model.py

```diff
@@ -25,6 +24,11 @@ class ModelGetDetailed(ModelGet):
     ...
 
 
+class ModelGetOverview(ModelBase):
+    description: str = ""
+    pipeline_count: int = 0
```

Comment:
I'd be tempted to remove the default. This value is a big part of why this schema exists, so I'd like the creator of the instance to be forced to fill it in themselves.